lib/preview/constants.js

```diff
@@ -102,6 +102,14 @@ module.exports = {
     ],
     'PDF': [
       'application/pdf'
+    ],
+    'VIDEO': [
+      'video/mp4',
+      'video/mpeg',
+      'video/quicktime',
+      'video/x-m4v',
+      'video/x-ms-wmv',
+      'video/x-msvideo'
```

Comment:
@johncrossman Apparently not... see https://jira.ets.berkeley.edu/jira/browse/COL-460 for what's been uploaded in the wild to date.